app/frontend/src/features/dashboard/Home.tsx

```diff
@@ -0,0 +1,146 @@
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  DialogContent,
+  makeStyles,
+  Typography,
+} from "@material-ui/core";
+import Button from "components/Button";
+import { Dialog, DialogActions, DialogTitle } from "components/Dialog";
+import { ExpandMoreIcon } from "components/Icons";
+import Markdown from "components/Markdown";
+import PageTitle from "components/PageTitle";
+import { CLOSE } from "features/constants";
+import {
+  ALL_COMMUNITIES_HEADING,
+  ALL_COMMUNITIES_LINK,
+  LANDING_TEXT,
+  LAST_UPDATE,
+  UPDATES_MARKDOWN,
+  UPDATES_TITLE,
+  WEEKLY_EVENTS_MARKDOWN,
+  WEEKLY_EVENTS_SUBTITLE,
+  WEEKLY_EVENTS_TITLE,
+  WELCOME,
+  YOUR_COMMUNITIES_HEADING,
+} from "features/dashboard/constants";
+import DashboardBanners from "features/dashboard/DashboardBanners";
+import React, { useState } from "react";
+
+import ContributorForm, { CONTRIBUTE, JOIN_THE_TEAM } from "../contributorForm";
+import CommunitiesList from "./CommunitiesList";
+
+const useStyles = makeStyles((theme) => ({
+  button: { display: "block", marginTop: theme.spacing(1) },
+  divider: { marginTop: theme.spacing(3) },
+  accordion: {
+    marginBlockStart: theme.spacing(2),
+    "& .MuiAccordionSummary-content": {
+      display: "grid",
+      alignItems: "baseline",
+      gridTemplateColumns: "repeat(auto-fit, 100%)",
+      [theme.breakpoints.up("md")]: {
+        gridTemplateColumns: "30vw 1fr",
```

Comment:
```suggestion
        gridTemplateColumns: "1fr 1fr",
```
Is there a good reason not to use `fr` unit for the title too? It looks alright to me when I played around in the dev tools, then it's less "magic value"

Reply:
But nice to see grid fixed this generally without the awkward `flexGrow: 100` thing!

Reply:
I want the title to be narrower. I'll do `1fr 2fr`